dotcom-rendering/src/components/AdSlot.tsx

```diff
@@ -622,33 +627,35 @@ export const AdSlot = ({
 		case 'fronts-banner': {
 			const advertId = `fronts-banner-${index}`;
 			return (
-				<div
-					className="ad-slot-container"
-					css={[
-						adContainerStyles,
-						frontsBannerAdContainerStyles,
-						hasPageskin && frontsBannerCollapseStyles,
-					]}
-				>
+				<div css={frontsBannerAdTopContainerStyles}>
```

Comment:
Would this also work if we applied the constraint to the ad slot, and the 'top' container styles moved to the existing container? If that is equivalent then we don't need an additional container.

Reply:
Will give this a go

Reply:
Ah, from testing, it seems that the `max-width` constraint needs to be on the ad slot container to communicate with DFP that fluid ads > 1300px can't go in this slot. If the constraint is on the ad slot, we see that fluid ads larger than 1300px can come through.

The container of the `fronts-banner` ad container is the `main` element of the page, so we can't change the styles here.

Reply:
Okay that makes sense thanks for checking!